website/docs/API.md

```diff
@@ -803,6 +805,15 @@ This option is also available as `defaultHidden` alias for compatibility with [R
 
 Default [debug options](#debug) to be used when calling `debug()`. These default options will be overridden by the ones you specify directly when calling `debug()`.
 
+
+#### `useLegacyQueries` option
+
+`byText`, `byPlaceholderText` and `byDisplayValue` queries used to return composite elements (see the doc on [testing environment](./TestingEnvironment.md)) for more information on composite and host components) but they now return host components. This can break tests if you use the element returned by these queries, for instance the props may be slightly different. To keep the legacy behavior, set this option to true. 
+
+#### `hostComponentNames` option
+
+The internal names of `Text` and `TextInput` components used by `byText`, `byDisplayValue` and `byPlaceholderText` queries. This should only be changed if you're using a version of React Native that is not compatible with the version of @testing-library/react-native. If that's the case, then failed getBy queries should suggest the appropriate config to use.
```

Comment:
```suggestion
The internal names of `Text` and `TextInput` components used by `byText`, `byDisplayValue` and `byPlaceholderText` queries. This should only be changed if you're using a version of React Native that is not compatible with the version of @testing-library/react-native. If that's the case, then failed getBy and findBy queries should suggest the appropriate config to use.
```

Reply:
I like having components before but I don't see much value in adding the precision on host here, wdyt ?

Reply:
Not that important, it's just so that people don't confuse it with composite components which don't have editable names. But then it's written in your paragraph title so I guess it's fine